NEWS

```diff
@@ -7,13 +7,177 @@
 ##
 #
 
-## Current
+## 1.12.0 RC1 (December 2, 2021)
 ### Features:
+#### Core
+* Added initial support for Go language bindings
```

Comment:
remove "initial"

Reply:
@yosefe Do we position as a preview or we can claim release. If this is release, I think we should move it to the section of its own, like Java

Reply:
how about
Added beta-level support for Go language bindings

Reply:
+1